hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -474,6 +474,11 @@ public class S3AFileSystem extends FileSystem implements StreamCapabilities,
    */
   private boolean fipsEnabled;
 
+  /**
+   * Is CRT enabled?
+   */
+  private boolean isCRTEnabled;
```

Comment:
let's not add another boolean field here.

In #7463 I've drafted a service under S3AStore which loads and stores config info, using an enumset for the flags. I'd really like to do reflection based loading of values, the way abfs does. Anyone want to pick up that work?

Reply:
for now, let's have that bool...config changes can follow.